mssql-config.yml.sample

```diff
@@ -35,6 +35,8 @@ integrations:
   labels:
     env: production
     role: mssql
+    # the label below is so there can be a common attribute to filter on within the dashboard. Most useful for multi-server environments
+    db_hostname: Enter-Custom-Hostname-Here    
```

Comment:
Why the hostname already included is not enough? Is it because it could refer to the host where the integration is running and not where the db is?

If we need something more stable I would add it through the args and not leveraging custom labels, it looks like a workaround

Moreover, it is possible that two DBs are running on the same host

Reply:
Hey @paologallinaharbur ,
The hostname itself is usually not enough or too difficult to understand due to the situations below:

1. The customers are running this OHI on a single host and monitoring other database instances remotely. This is a common use case for larger environments. In those cases the hostname is the name of the host where the OHI is running, which isn't useful when querying.
2. The entityName isn't a good substitute for the situation above due to managed services (RDS, Azure managed instances) being given random names. 
3. Even if running on the host, the infra friendly hostname doesn't get decorated with the OHI metrics/events. The hostname for the OHI metrics is the default EC2 hostname, which again, isn't necessarily readable/usable.

The db_hostname attribute allows customers to give the instance a friendlier name. I'm open to using other attributes, but I'm running this OHI in the manner above to mimic how our customers are running it and that's what I'm seeing in the data.